### PR TITLE
Add argument parameter for Launch Application process keyword #23

### PR DESCRIPTION
**Pull request by Nepitwin**

### CODE REVIEW

File: TestApplications/src/Notifier/Form1.cs [hunk 1]
```diff
@@ -3,19 +3,24 @@
...
-    public partial class Notifier : Form
+    public sealed partial class Notifier : Form
     {
-        public Notifier()
+        public Notifier(string title = "Notifier", bool disableTimer = false)
         {
             InitializeComponent();
 
-            var aTimer = new Timer
+            Text = title;
```
Comment (noubar) [inline TestApplications/src/Notifier/Form1.cs:12]: where or what for is this text variable used?

Comment (noubar) [reply TestApplications/src/Notifier/Form1.cs:12]: ok i see this should represent the title of the app

**Review by noubar — APPROVED** [review]

File: TestApplications/src/Notifier/Form1.cs (hunk 1 above)
Comment (Nepitwin) [reply TestApplications/src/Notifier/Form1.cs:12]: The test will change the title bar from the application :)

**Comment by noubar** [comment]
a side comment i see that as well in docu as in code the second arg of the lunch app is called arguments with s. Despite of the fact that it can not take more than one argument and forward it to the exe.

**Comment by Nepitwin** [comment]
> a side comment i see that as well in docu as in code the second arg of the lunch app is called arguments with s. Despite of the fact that it can not take more than one argument and forward it to the exe.

It can take more than one argument as a string representation. See FlaUI arguments in Application.cs

https://github.com/FlaUI/FlaUI/blob/master/src/FlaUI.Core/Application.cs

   --> public static Application Launch(string executable, string arguments = null)

**Pull request merged**